index.html

```diff
@@ -99,7 +99,8 @@ <h2>About me</h2>
         <div>
           <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         </div>
-        <form action="https://formspree.io/f/mgedpkpr" method="POST">
+        <form id="form" action="https://formspree.io/f/mgedpkpr" method="POST">
+          <div class="form-error"></div>
```

Comment:
- [ ] Kindly be noticed that the error message should show up close to the `submit` button.